packages/suite/src/components/wallet/CoinjoinSummary/AccountSummarySection/index.tsx

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
 import styled from 'styled-components';
-import { AccountSummaryHeader } from './AccountSummaryHeader';
+
+import { H3 } from '@trezor/components';
+import { Translation } from '@suite-components/Translation';
 import { AnonymityChart } from './AnonymityChart';
+import { AnonymityIndicator } from './AnonymityIndicator';
 import { BalanceSection } from './BalanceSection';
 
 const Container = styled.div`
     width: 100%;
 `;
 
-interface AccountSummaryProps {
-    onAnonymize: () => void;
-}
+const Row = styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 28px;
+`;
+
+const StyledAnonymityIndicator = styled(AnonymityIndicator)`
+    margin-left: auto;
+`;
 
-export const AccountSummarySection = ({ onAnonymize }: AccountSummaryProps) => (
+export const AccountSummarySection = () => (
     <Container>
-        <AccountSummaryHeader />
+        <Row>
```

Comment:
Should that also be `WalletLayoutHeader` but without the close button?

Reply:
I separated these two because the heading on this page is smaller than i `WalletLayoutHeader` and I would also have to add a prop to the `WalletLayoutHeader` for (not) displaying the close button.

Reply:
Didn't notice that it's smaller :)